gossip/src/crds_value.rs

```diff
@@ -566,6 +571,9 @@ impl RawOffsets {
     }
 }
 
+// Per design doc, we should start wen_restart within 9 hours.
+pub const MAX_SLOTS_RESTART_LAST_VOTED_FORK_SLOTS: usize = 81000;
```

Comment:
This definition also needs to go inside `struct RestartLastVotedForkSlots`. Similar to `MAX_BYTES`.

With 5000 nodes, each sending 81k slot, it would take 3GB of memory just to store all those `Vec<Slot>`.
Can we reduce this limit to `65,535` so that we can stick with `u16`?

Reply:
I think that's fine. The original argument for 9 hours is so that admins can wake up and join the restart, but if they sleep sound enough then even 9 hours wouldn't be enough. In reality we normally have >80% of the admins wake up within 7 hours.